gcc/rust/backend/rust-compile-var-decl.h

```diff
@@ -30,39 +30,42 @@ class CompileVarDecl : public HIRCompileBase, public HIR::HIRPatternVisitor
   using HIR::HIRPatternVisitor::visit;
 
 public:
-  static ::Bvariable *compile (tree fndecl, tree translated_type,
-			       HIR::Pattern *pattern, Context *ctx)
+  static void compile (tree fndecl, tree translated_type, HIR::Pattern *pattern,
+		       std::vector<Bvariable *> *locals, Context *ctx)
```

Comment:
How about passing a reference here?
```suggestion
		       std::vector<Bvariable *> &locals, Context *ctx)
```